pylearn2/sandbox/rnn/space/__init__.py

```diff
@@ -69,16 +69,10 @@ def __str__(self):
     @wraps(space.Space._format_as_impl)
     def _format_as_impl(self, is_numeric, batch, space):
         assert isinstance(space, SequenceSpace)
-        if is_numeric:
-            rval = np.apply_over_axes(
-                lambda batch, axis: self.space._format_as_impl(
-                    is_numeric=is_numeric,
-                    batch=batch,
-                    space=space.space),
-                batch, 0)
-        else:
-            NotImplementedError("Can't convert SequenceSpace Theano variables")
```

Comment:
I'm not confident in the implementation of `SequenceDataSpace._format_as_impl` with `is_numeric=False`, maybe it is why it was disabled here.